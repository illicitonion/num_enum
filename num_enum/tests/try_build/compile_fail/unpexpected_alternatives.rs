#[derive(num_enum::UnsafeFromPrimitive)]
#[repr(u8)]
enum Numbers {
    Zero,
    #[num_enum(alternatives = [2])]
    One,
}

fn main() {

}
