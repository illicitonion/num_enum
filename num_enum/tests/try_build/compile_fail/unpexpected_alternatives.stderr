error: #[derive(UnsafeFromPrimitive)] does not support `#[num_enum(alternatives = [..])]`
 --> $DIR/unpexpected_alternatives.rs:5:16
  |
5 |     #[num_enum(alternatives = [2])]
  |                ^^^^^^^^^^^^
